src/conf_mode/high-availability.py

```diff
@@ -51,6 +51,8 @@ def get_config(config=None):
     if 'vrrp' in ha:
         if 'group' in ha['vrrp']:
             default_values_vrrp = defaults(base_vrrp + ['group'])
+            if 'garp' in default_values_vrrp:
```

Comment:
Won't this statement always return true and delete garp default value, even if the garp node is defined within a group?